README.md

```diff
@@ -1,555 +1,242 @@
+receive the results (which prevents doing work when the results are never used).
+That’s easy enough:
+
+```swift
+searchResults.start(next: { results in
+    println("Search results: \(results)")
+}, error: { error in
+    println("Error searching: \(error)")
+})
 ```
 
-That demonstrates some of what RAC can do, but it doesn't demonstrate why RAC is
-so powerful. It's hard to appreciate RAC from README-sized examples, but it
-makes it possible to write code with less state, less boilerplate, better code
-locality, and better expression of intent.
+Here, we watch for the `Next` and `Error`
+[events](Documentation/FrameworkOverview.md#events), and just log to the
+console. This could easily do something else instead, like update a table view
+or a label on screen.
 
-For more sample code, check out [C-41][] or [GroceryList][], which are real iOS
-apps written using ReactiveCocoa. Additional information about RAC can be found
-in the [Documentation][] folder.
+#### Throttling requests
 
-## When to use ReactiveCocoa
+Now, let’s say you only want to actually perform the search when the user pauses
+typing, to minimize traffic.
 
-Upon first glance, ReactiveCocoa is very abstract, and it can be difficult to
-understand how to apply it to concrete problems.
+ReactiveCocoa has a declarative `throttle` operator that we can apply to our
+search strings:
 
-Here are some of the use cases that RAC excels at.
+```swift
+let searchStrings = textField.rac_textSignal()
+    |> toSignalProducer()
+    |> map { text in text as! String }
+    |> throttle(0.5, onScheduler: UIScheduler())
+```
 
-### Handling asynchronous or event-driven data sources
+This prevents values from being sent less than 0.5 seconds apart, so the user
+must stop editing for at least that long before we’ll use their search string.
 
-Much of Cocoa programming is focused on reacting to user events or changes in
-application state. Code that deals with such events can quickly become very
-complex and spaghetti-like, with lots of callbacks and state variables to handle
-ordering issues.
+To do this manually would require significant state, and end up much harder to
+read! With ReactiveCocoa, we can use just one operator to incorporate _time_ into
+our event stream.
 
-Patterns that seem superficially different, like UI callbacks, network
-responses, and KVO notifications, actually have a lot in common. [RACSignal][]
-unifies all these different APIs so that they can be composed together and
-manipulated in the same way.
+## Objective-C and Swift
 
-For example, the following code:
+Although ReactiveCocoa was started as an Objective-C framework, as of version
+3.0, all major feature development is concentrated on the Swift API.
 
-```objc
+RAC’s Objective-C API and Swift API are entirely separate, but there is
+a [bridge](Documentation/ObjectiveCBridging.md) to convert between the two. This
+is mostly meant as a compatibility layer for older ReactiveCocoa projects, or to
```

Comment:
What do you think about linking to the old Obj-C docs from here? I imagine some people will land at this README because they got some RAC 2 code they're maintaining.